src/de/emodelle.html

```diff
@@ -1,5 +1,15 @@
-
-      <%= require('html-loader!./components/02_footer_section/index.html').default %>
-
-      <br>
-      THIS VARIABLE IS FROM EMODELLE:   <%= htmlWebpackPlugin.options.newVariable %>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+      <head>
+            <!-- relative path to SCSS source file -->
+            <link href="./components/02_footer_section/style.scss" rel="stylesheet" />
+            <!-- relative path to JS source file -->
+            <script src="../js/app.js" defer="defer"></script>
```

Comment:
you can define source script file directly in the HTML template using the `script` tag and the path to source JS file